.env.prod

```diff
@@ -11,6 +11,7 @@ REACT_APP_INDEX_TOKEN_ADDRESS=0x0954906da0Bf32d5479e25f46056d22f08464cab
 REACT_APP_DAI_TOKEN_ADDRESS=0x6B175474E89094C44Da98b954EedeAC495271d0F
 REACT_APP_USDC_TOKEN_ADDRESS=0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
 REACT_APP_BTC2X_FLI_TOKEN_ADDRESS=0x0B498ff89709d3838a063f1dFA463091F9801c2b
+REACT_APP_ETH_TOKEN_ADDRESS=0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
```

Comment:
Is this canonical? Or is this specific to 0x's API?

Reply:
It seems like this address is only used in relation to 0x API. If it's not canonical we should label it as such. Maybe include it as a constant in `ethContractAddresses` instead of setting it as ENV